### ISSUE TITLE
Also maintain v19 port

### ISSUE THREAD

**Issue by mvysny** [mvysny/vaadin-quarkus#14]
Bring the v18 branch up-to-speed with the main branch (which tracks Vaadin 14).

Blocked by #6 since it would be good to have everything merged into main before merging into v18

**Comment by mvysny** [comment]
Also blocked by #11 since we can then demo Lit both in the main branch and in the v19 branch.

We should actually use Vaadin 19 since there has been a change in how Vaadin 19 loads resources (the `Lookup` API). That will also test the Quarkus+Vaadin Extension integration.

**Comment by mvysny** [comment]
Vaadin 14.5 will only be released next month, which is too late. I'll only implement the lit demo for v19.

**Comment by mvysny** [comment]
Initial v19 port is now residing in the v19 branch.

**Comment by mvysny** [comment]
The v19 branch now compiles, tests are running properly and the app is also running properly. Closing as fixed. Fixed in 5baf41a2b784f38af952e61bfcf8bb9f169854ad
